src/lsp.rs

```diff
@@ -151,6 +189,20 @@ impl<TLocalLinter: LocalLinter> Backend<TLocalLinter> {
                 .unwrap();
         }
     }
+
+    fn start_new_trace(&self, trace_name: &str) {
+        if let Some(start_new_trace_sender) = self.start_new_trace_sender.as_ref() {
+            start_new_trace_sender
+                .send(
+                    format!(
+                        "/Users/jrosse/prj/hello-world/trace-{trace_name}-{}.json",
```

Comment:
Can figure out how to be aware of eg project directory at some point but for now the bar is "I can use it locally"